src/app/components/pages/Witnesses.jsx

```diff
@@ -76,6 +78,10 @@ class Witnesses extends React.Component {
             const owner = item.get('owner');
             const thread = item.get('url');
             const myVote = witness_votes ? witness_votes.has(owner) : null;
+            const signingKey = item.get('signing_key');
+            const witnessIsDisabled = /STM1111111111111111111111111111111114T1Anm/.test(signingKey);
+            const priceFeed = item.get('sbd_exchange_rate');
+            const noPriceFeed = /0.000 STEEM/.test(priceFeed.get('base'));
```

Comment:
and 0.000 STEEM - also, is this strict enough? If this is a match over the entire string choose the faster of 
1. a good regex
2. ===